client/my-sites/media-library/test/list.jsx

```diff
@@ -55,7 +51,8 @@ describe( 'MediaLibraryList item selection', function() {
 			data: fixtures
 		} );
 
-		mockery.registerMock( './list-item', EMPTY_COMPONENT );
+		mockery.registerMock( 'components/infinite-list', EmptyComponent );
```

Comment:
`<InfiniteList />` have async callbacks which cause issues. We don't verify it's logic here so I mocked it out :)